tests/test_language_extract_rust.py

```diff
@@ -80,3 +80,21 @@ def test_push(request, string, offset, encoding, rust_strings):
 )
 def test_mov_jmp(request, string, offset, encoding, rust_strings):
     assert StaticString(string=string, offset=offset, encoding=encoding) in request.getfixturevalue(rust_strings)
+
+
+@pytest.mark.parametrize(
+    "string,offset,encoding,rust_strings",
+    [
+        # .rdata:004BFA48                 dd offset unk_4BA13A
+        # .rdata:004BFA4C                 dd offset unk_4BA100
+        pytest.param("Invalid branch target in DWARF expression", 0xB813A, StringEncoding.UTF8, "rust_strings32"),
+        pytest.param(
+            "Expected to find an FDE pointer, but found a CIE pointer instead.",
+            0xB8163,
+            StringEncoding.UTF8,
+            "rust_strings32",
+        ),
+    ],
+)
+def test_raw_xrefs(request, string, offset, encoding, rust_strings):
```

Comment:
i think its good that we have integration tests that show the whole system working together to find the strings. I think we should also have some tests for the specific routines that you added, so we can verify their behavior directly. something like `test_get_raw_xrefs`.

Reply:
Hi, what should be the best approach to that? As there are not just `test_get_raw_xrefs`, but also others such as `find_i386_push_xrefs`, `find_lea_xrefs`, etc., should we test them too in a separate file or another PR? What are your thoughts?